### ISSUE TITLE
Dropdown filters need programmatic name or label

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2724]
Found via Trusted Tester -  violation exists on all pages with drop down filters: 
https://revenuedata.doi.gov/
https://revenuedata.doi.gov/explore
https://revenuedata.doi.gov/query-data

Example: ![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/62c3db30-e6b1-469a-85a2-9ec05943bce0)

**Comment by lpgoldstein** [comment]
No longer relevant
